ports/servoshell/prefs.rs

```diff
@@ -781,18 +765,11 @@ fn print_debug_options_usage(app: &str) {
         "relayout-event",
         "Print notifications when there is a relayout.",
     );
-    print_option(
-        "show-fragment-borders",
-        "Paint borders along fragment boundaries.",
-    );
-    print_option(
-        "show-parallel-layout",
-        "Mark which thread laid each flow out with colors.",
-    );
     print_option(
         "signpost",
         "Emit native OS signposts for profile events (currently macOS only)",
     );
+    print_option("dump-style-stats", "Print style stats after each restyle.");
```

Comment:
Any reason to move this here, instead of having it grouped with the other "dump" options?